tests/array.mjs

```diff
@@ -3,7 +3,8 @@ import {
   formatList,
   isArray,
   rejectFromArray,
-  selectFromArray
+  selectFromArray,
+  uniqueArray
```

Comment:
Wonder if the name of this filter needs to be a little clearer, maybe `uniqueFromArray` or `uniqueInArray`?

Reply:
Yes, good point. Either of those make sense to me. uniqueFromArray I think makes slightly more sense to me. If i had to choose.

Reply:
I've updated it to `uniqueFromArray`